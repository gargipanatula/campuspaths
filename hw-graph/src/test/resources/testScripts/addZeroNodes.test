# Create a graph
CreateGraph graph1

# Print the nodes in graph1
ListNodes graph1