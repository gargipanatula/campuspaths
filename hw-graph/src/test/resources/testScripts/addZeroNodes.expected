# Create a graph
created graph graph1

# Print the nodes in graph1
graph1 contains: